recipe/VS2019.patch

```diff
@@ -0,0 +1,47 @@
+diff --git a/example/CMakeLists.txt b/example/CMakeLists.txt
+index fe6c2a77..75499316 100644
+--- a/example/CMakeLists.txt
++++ b/example/CMakeLists.txt
+@@ -27,7 +27,7 @@ if (APPLE)
+ elseif (UNIX)
+     string(CONCAT ${MASTER_PROJECT_NAME}_LIB_NAME "lib" ${${MASTER_PROJECT_NAME}_LIB_NAME} ".so")
+ elseif(_WIN32)
+-    message(FATAL_ERROR "Examples are not implemented for Windows")
++    message("TO BE TESTED")
+     string(CONCAT ${MASTER_PROJECT_NAME}_LIB_NAME ${${MASTER_PROJECT_NAME}_LIB_NAME} ".dll")
+ endif()
+ set(${MASTER_PROJECT_NAME}_LIBRARY ${CMAKE_BINARY_DIR}/${${MASTER_PROJECT_NAME}_LIB_NAME})
+diff --git a/include/biorbdConfig.h.in b/include/biorbdConfig.h.in
+index 3003bd7c..4998039f 100644
+--- a/include/biorbdConfig.h.in
++++ b/include/biorbdConfig.h.in
+@@ -12,10 +12,6 @@
+ //#  define BIORBD_API
+ //#endif
+ 
+-#ifdef _WIN32
+-#define _ENABLE_EXTENDED_ALIGNED_STORAGE
+-#endif
+-
```

Comment:
This patch doesn't add compiler definitions, it tries to remove them.

Reply:
I am not sure what you mean.. 
The Aligment problem is a well known problem for code that are ported from <VS2019 to >=VS2019. The compiler will fail as a warning so the programmer chooses if they want _ENABLE_EXTENDED_ALIGNED_STORAGE or not (another define). When one of the defined is choosen, then the compiler is supposed to compile properly.

For some reason, the compiler used here doesn't recognized this is actually defined (while if compiling locally it does).

Reply:
The patch is a git diff, so lines that start with "-" are subtracted and lines starting with "+" are added. These lines start with "-". So the diff instruction is to remove these compiler definitions not add them.

```
diff --git a/include/biorbdConfig.h.in b/include/biorbdConfig.h.in
index 3003bd7c..4998039f 100644
--- a/include/biorbdConfig.h.in
+++ b/include/biorbdConfig.h.in
@@ -12,10 +12,6 @@
 //#  define BIORBD_API
 //#endif
 
-#ifdef _WIN32
-#define _ENABLE_EXTENDED_ALIGNED_STORAGE
-#endif
-
 #define BIORBD_API
 #cmakedefine IPOPT_FOUND
```

Reply:
I probably used git diff on reverse.. I will reulpload the file

Reply:
I reuploaded with
`git diff 229c17bad31fbc8dd357f903185e74dcf51e06e2  36f3578a2506177e5621f3d7eca85606cb4ccba1 > VS2019.patch`